static/app/views/issueList/addViewPage.tsx

```diff
@@ -22,6 +22,7 @@ import {NewTabContext} from 'sentry/views/issueList/utils/newTabContext';
 type SearchSuggestion = {
   label: string;
   query: string;
+  scope?: 'personal' | 'organization';
```

Comment:
Is there a reason to make this optional?

Reply:
@malwilley yeah this is used by the suggestionlist component, which the recommended searches list also uses, and those do not have an explicit scope attached